specifications/raml/api.raml

```diff
@@ -1,94 +1,29 @@
+uses:
+  objectModels: !include libraries/objectModels/objects.raml
+types: !include types/types.raml
 resourceTypes:
-  - base:
-      get?: &common
-        headers:
-          Accept:
-            description: It is used to set specified media type.
-            type: string
-        responses:
-          429:
-            description: |
-              API rate limit exceeded. See http://developer.spotify.com/web-api/user-guide/#rate-limiting for details.
-      post?: *common
-      put?: *common
-      delete?: *common
-  - item:
-      type: base
-      get?:
-  - collection:
-      type: base
-      get?:
-securitySchemes:
-  - oauth_2_0:
-      description: |
-          Spotify supports OAuth 2.0 for authenticating all API requests.
-      type: OAuth 2.0
-      describedBy:
-        headers:
-          Authorization:
-            description: |
-              Used to send a valid OAuth 2 access token.
-            type: string
-        responses:
-          401:
-            description: |
-              Bad or expired token. This can happen if the user revoked a token or
-              the access token has expired. You should re-authenticate the user.
-          403:
-            description: |
-              Bad OAuth request (access token without necessary scope).
-      settings:
-        authorizationUri: https://accounts.spotify.com/authorize
-        accessTokenUri: https://accounts.spotify.com/api/token
-        authorizationGrants: [ code, token ]
-        scopes:
-          - "user-library-read"
-          - "user-library-modify"
-          - "user-read-private"
-          - "user-read-birthdate"
-          - "user-read-email"
-          - "user-follow-read"
-          - "user-follow-modify"
-          - "playlist-read-collaborative"
-          - "playlist-read-private"
-          - "playlist-modify-public"
-          - "playlist-modify-private"
+  - base: !include resourceTypes/base.raml
+  - item: !include resourceTypes/item.raml
+  - collection: !include resourceTypes/collection.raml
 traits:
-  - filterable:
-      queryParameters:
-        fields:
-          description: A comma-separated list of fields to filter query
-          type: string
-          example: items(added_by.id,track(name,href,album(name,href)))
-          displayName: Fields
-  - paged:
-      queryParameters:
-        limit:
-          displayName: Limit
-          description: The maximum number of items to return
-          type: integer
-          example: 10
-          minimum: 0
-          default: 20
-          maximum: 50
-          required: false
-        offset:
-          displayName: Offset
-          description: The index of the first item to return
-          type: integer
-          example: 5
-          default: 0
-          required: false
+  - filterable: !include traits/filterable.raml
+  - paged: !include traits/paged.raml
+securitySchemes:
+  - oauth_2_0: !include securitySchemes/oauth_2_0.raml
```

Comment:
`traits:` and `securitySchemas` should be maps, means, the hyphens should be removed.